js/partials.js

```diff
@@ -1,3 +1,45 @@
+const RubySummit = {
+  renderSchedule: function() {
+    fetch("data/schedule.json")
+    .then(response => {
+      return response.json()
+    })
+    .then(schedule => {
+      schedule.map(track => {
+        const trackDay = document.querySelector(`.track[data-day="${track.day}"]`)
+
+        const table = trackDay.querySelector('tbody')
+        track.talks.forEach(talk => {
+          name = talk.name ? `<br /> ${talk.name}` : ''
+
+          table.innerHTML +=
+            `
+        <tr class="talk-info">
+        <td>
+          <span title="${talk.hour}"><strong>${talk.hour}</strong></span>
+        </td>
+        <td>
+          <span class="title" title="${talk.title}"><strong>${talk.title}</strong></span>
+          <span class="speaker-name" title="${name}">${name}</span>
```

Comment:
![image](https://user-images.githubusercontent.com/1077589/99831077-5391de00-2b3d-11eb-8cd1-86db9a93bd23.png)

```suggestion
          table.innerHTML +=
            `
        <tr class="talk-info">
        <td>
          <span title="${talk.hour}"><strong>${talk.hour}</strong></span>
        </td>
        <td>
          <span class="title" title="${talk.title}"><strong>${talk.title}</strong></span>
          <span class="speaker-name" title="${talk.name}"><br /> ${talk.name}</span>
```

Reply:
Mesmo caso que acima :)